Example/Tests/Tests.m

```diff
@@ -15,13 +15,26 @@ @interface SEGMockADBMediaHeartbeatFactory : NSObject <SEGADBMediaHeartbeatFacto
 @end
 
 
+@interface SEGMockADBMediaObjectFactory : NSObject <SEGADBMediaObjectFactory>
```

Comment:
FWIW didn't really need to create a new type here. You can use OCMockito to mock the protocol.